src/factotum_server/mod.rs

```diff
@@ -83,6 +84,7 @@ pub fn start(args: Args) -> Result<(), String> {
         status:     get     "/status"   =>  responder::status,
         settings:   post    "/settings" =>  responder::settings,
         submit:     post    "/submit"   =>  responder::submit,
+        submit_new:     post    "/submit/new"   =>  responder::submit,
```

Comment:
why do we need a `new` route?

Reply:
Sometimes the same task wants to execute multiple times, and you need to know the results of the run.